tests/test_relative_free_energy.py

```diff
@@ -183,6 +179,55 @@ def test_steps_per_frames():
         np.testing.assert_array_equal(frame[-1], test_frame[-1])
 
 
+def test_imaging_frames():
+    """Should not be able to run a relative free energy calculation with a single window"""
```

Comment:
Is this docstring intended? I don't immediately understand how this relates to the test.

Reply:
Nope, copy pasta

Reply:
Changed in [3b58a0b](https://github.com/proteneer/timemachine/pull/905/commits/3b58a0ba366ae6c2d9a7da2446961aa52271ccee)